src/script/interpreter.h

```diff
@@ -332,6 +316,18 @@ class DeferringSignatureChecker : public BaseSignatureChecker
     }
 };
 
+/** Dummy signature checker which accepts all signatures. */
+class DummySignatureChecker final : public BaseSignatureChecker
+{
+public:
+    DummySignatureChecker() {}
+    bool CheckECDSASignature(const std::vector<unsigned char>& scriptSig, const std::vector<unsigned char>& vchPubKey, const CScript& scriptCode, SigVersion sigversion) const override { return true; }
+    bool CheckSchnorrSignature(Span<const unsigned char> sig, Span<const unsigned char> pubkey, SigVersion sigversion, ScriptExecutionData& execdata, ScriptError* serror) const override { return true; }
+    bool CheckLockTime(const CScriptNum& nLockTime) const override { return false; }
+    bool CheckSequence(const CScriptNum& nSequence) const override { return false; }
+};
+const DummySignatureChecker DUMMY_CHECKER;
```

Comment:
Add `inline` keyword to this variable defn in a header?  I know this class doesn't have any data members - in fact all it has is a virtual table pointer.  So the fact that it (right now) has internal linkage and is thus duplicated wherever `interpreter.h` is included doesn't cost much.  At all.  But, maybe its better to mark it properly.